contracts/test/LightClient.t.sol

```diff
@@ -369,45 +353,41 @@ contract LightClient_newFinalizedState_Test is LightClientCommonTest {
-                // first block of a new epoch (from epoch 2) should update the following
-                assertEq(lc.currentEpoch(), 2);
-                bytes32 genesisComm = lc.computeStakeTableComm(genesis);
-                LC.LightClientState memory lastBlockInFirstEpoch = states[i - 1];
-                bytes32 firstEpochComm = lc.computeStakeTableComm(lastBlockInFirstEpoch);
-                assertEq(lc.votingStakeTableCommitment(), genesisComm);
-                assertEq(lc.frozenStakeTableCommitment(), firstEpochComm);
-                assertEq(lc.votingThreshold(), genesis.threshold);
-                assertEq(lc.frozenThreshold(), lastBlockInFirstEpoch.threshold);
-            } else {
-                assertEq(lc.currentEpoch(), 1);
-                bytes32 stakeTableComm = lc.computeStakeTableComm(genesis);
-                assertEq(lc.votingStakeTableCommitment(), stakeTableComm);
-                assertEq(lc.frozenStakeTableCommitment(), stakeTableComm);
-                assertEq(lc.votingThreshold(), genesis.threshold);
-                assertEq(lc.frozenThreshold(), genesis.threshold);
-            }
+            //            if (i == statesLen - 1) {
+            //                // first block of a new epoch (from epoch 2) should update the
+            // following
+            //                bytes32 genesisComm = lc.computeStakeTableComm(genesis);
+            //                LC.LightClientState memory lastBlockInFirstEpoch = states[i - 1];
+            //                bytes32 firstEpochComm =
+            // lc.computeStakeTableComm(lastBlockInFirstEpoch);
+            //                assertEq(lc.votingStakeTableCommitment(), genesisComm);
+            //                assertEq(lc.frozenStakeTableCommitment(), firstEpochComm);
+            //                assertEq(lc.votingThreshold(), genesis.threshold);
+            //                assertEq(lc.frozenThreshold(), lastBlockInFirstEpoch.threshold);
+            //            } else {
+            bytes32 stakeTableComm = lc.computeStakeTableComm(genesis);
+            assertEq(lc.votingStakeTableCommitment(), stakeTableComm);
+            assertEq(lc.frozenStakeTableCommitment(), stakeTableComm);
+            assertEq(lc.votingThreshold(), genesis.threshold);
+            assertEq(lc.frozenThreshold(), genesis.threshold);
+            //}
         }
     }
 
     /// @dev Test happy path for updating after skipping a few blocks (but not an epoch)
     /// forge-config: default.fuzz.runs = 4
     /// forge-config: quick.fuzz.runs = 1
     /// forge-config: ci.fuzz.runs = 10
-    function test_UpdateAfterSkippedBlocks(uint32 numBlockSkipped, uint32 numBlockPerEpoch)
-        external
-    {
-        numBlockPerEpoch = uint32(bound(numBlockPerEpoch, 2, 10));
-        numBlockSkipped = uint32(bound(numBlockSkipped, 1, numBlockPerEpoch - 1));
+    function test_UpdateAfterSkippedBlocks(uint32 numBlockSkipped) external {
```

Comment:
update the `@dev` code comment, remove the "(but not an epoch)" part.

Reply:
addressed here https://github.com/EspressoSystems/espresso-sequencer/commit/5acfd9472fc649eb6c8d6c76507789a318e55a99